[2024-10-16T09:56:37.768+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-07-22T00:00:00+00:00 [queued]>
[2024-10-16T09:56:37.785+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-07-22T00:00:00+00:00 [queued]>
[2024-10-16T09:56:37.785+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:56:37.786+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:56:37.786+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:56:37.808+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-07-22 00:00:00+00:00
[2024-10-16T09:56:37.814+0000] {standard_task_runner.py:55} INFO - Started process 9161 to run task
[2024-10-16T09:56:37.821+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-07-22T00:00:00+00:00', '--job-id', '1909', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmprtqtqhuw']
[2024-10-16T09:56:37.822+0000] {standard_task_runner.py:83} INFO - Job 1909: Subtask insert_rate
[2024-10-16T09:56:37.921+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-07-22T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:56:37.971+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:56:38.038+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-07-22T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-07-22T00:00:00+00:00
[2024-10-16T09:56:38.040+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 473.28139567, '2024-07-22')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:56:38.051+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:56:38.308+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:56:38.318+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 473.28139567, '2024-07-22')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:56:38.322+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:56:38.336+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240722T000000, start_date=20241016T095637, end_date=20241016T095638
[2024-10-16T09:56:38.369+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:56:38.403+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T10:06:44.587+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-07-22T00:00:00+00:00 [queued]>
[2024-10-16T10:06:44.618+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-07-22T00:00:00+00:00 [queued]>
[2024-10-16T10:06:44.620+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:06:44.621+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T10:06:44.623+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:06:44.833+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-07-22 00:00:00+00:00
[2024-10-16T10:06:44.849+0000] {standard_task_runner.py:55} INFO - Started process 11767 to run task
[2024-10-16T10:06:44.861+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-07-22T00:00:00+00:00', '--job-id', '2655', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmpdplnhff3']
[2024-10-16T10:06:44.863+0000] {standard_task_runner.py:83} INFO - Job 2655: Subtask insert_rate
[2024-10-16T10:06:45.174+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-07-22T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T10:06:45.341+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T10:06:45.546+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-07-22T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-07-22T00:00:00+00:00
[2024-10-16T10:06:45.551+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 473.28139567, '2024-07-22')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T10:06:45.585+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:06:46.461+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:06:46.493+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 473.28139567, '2024-07-22')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T10:06:46.501+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T10:06:46.555+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240722T000000, start_date=20241016T100644, end_date=20241016T100646
[2024-10-16T10:06:46.658+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T10:06:46.724+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
